ui/src/main/res/values-pt-rPT/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="payment_activity_toolbar_title">Faça o Pagamento</string>
+    <string name="payment_card_pay_button_text">Pague com o M-Pesa</string>
+    <string name="payment_card_view_phone_number_prompt">Insira o número de telefone da sua conta M-Pesa</string>
+    <string name="payment_authentication_card_title">Verifique o seu telefone</string>
+    <string name="payment_authentication_card_description">Uma notificação USSD foi enviada para o número: %s \n\nInsira o PIN do seu M-Pesa para completar a transação.</string>
+    <string name="payment_authentication_card_learn_more_button_text">Saiba mais</string>
+    <string name="payment_authentication_card_timer_seconds_text">Segundos</string>
+    <string name="payment_error_card_title">O Pagamento Falhou</string>
+    <string name="payment_error_card_retry_button">Tente Novamente</string>
+    <string name="payment_response_ins_0">Pagemento Concluído com Sucesso.</string>
+    <string name="payment_response_ins_unhandled_response">Algo deu errado ao processar o seu pagamento.</string>
+    <string name="payment_response_ins_2006">O pagamento falhou devido ao saldo insuficiente na sua conta.</string>
+    <string name="payment_response_ins_5">Pagemento cancelado. Tente novamente se quiser alterar o número da conta M-Pesa.</string>
+    <string name="payment_response_ins_4">Pagamento falhou. O número de telefone inserido não tem uma conta M-Pesa válida.</string>
+    <string name="payment_response_ins_9">"A solicitação do pagamento atingiu o tempo limite porque o PIN de autorização não foi insirido a tempo. "</string>
```

Comment:
~insirido~ inserido

Reply:
In this same line: Why do we have it between quotation marks? I don't see it on the english version.